README.md

```diff
@@ -193,6 +193,7 @@ Questions can be asked by raising an `Issue`.
 - [Josh Callahan](https://github.com/joshcallahan)
 - [Joykishan Sharma](https://github.com/JoykishanSharma)
 - [Kabir Jain](https://github.com/kkkkkabir)
+- [kuwar kapur](https://github.com/kuwarkapur)
 - [Kaiwalya Koparkar](https://github.com/kaiwalyakoparkar)
 - [Kehinde Adeleke](https://github.com/adeleke5140)
 - [Krishna R ](https://github.com/Krishna-Ravi)
```

Comment:
```suggestion
- [Krishna R ](https://github.com/Krishna-Ravi)
- [kuwar kapur](https://github.com/kuwarkapur)

Reply:
I suggest writing their names with CAPS (the first character of the first name and the first character of the second name).

```suggestion
- [Krishna R ](https://github.com/Krishna-Ravi)
- [Kuwar Kapur](https://github.com/kuwarkapur)
```